### ISSUE TITLE
Fix broken Nevow link in docs.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5883]
|<img alt="faulkner's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| faulkner reported|
|-|-|
|Trac ID|trac#5883|
|Type|defect|
|Created|2012-08-19 19:12:27Z|

Looks like this is the only one of the links that didn't get switched over to launchpad.

Attachments:

* [fix-nevow-link.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/81f/81f01597f9b8c274f8c4f7a4219061e0d69654f8/9df0ae765643a874a8daef718fd1a2a6f66b154c.patch) (781 bytes) - added by faulkner on 2012-08-19 19:12:36Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5883 5883
type__defect defect
reporter__faulkner faulkner
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1345403547000000 1345403547000000
changetime__1345570041000000 1345570041000000
version__None None
owner__thijs thijs

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks for your patch. Make sure to add the `review` keyword when you propose a patch as described on ReviewProcess#Authors:Howtogetyourchangereviewed

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [35326]) Apply fix-nevow-link.patch: Correct a link to the Nevow project, available from launchpad now.

Author: faulkner
Reviewer: thijs
Fixes: [#5883](https://github.com/twisted/twisted/issues/5883)
